src/features/wireshark/WiresharkButton.tsx

```diff
@@ -74,13 +74,8 @@ export default ({ extendedDescription = false }: WiresharkProps) => {
             ) : (
                 <>
                     <h6>Wireshark not detected</h6>
+                    {extendedDescription && <p> {extendedDescription}</p>}
```

Comment:
Because `extendedDescription` is a boolean, `<p> {extendedDescription}</p>` will be  (if `extendedDescription` is true) just an empty `<p></p>` tag. Is this correct?

Reply:
I agree, it would just be empty. I think that the entire property can be removed now, since no components makes a wireshark button with the extended description prop set to true anymore.

Reply:
I forgot about this one, I was actually meant to remove it. Thanks for pointing it out.

Reply:
Removed the property in 8dba1e9b7727f4bc54d04802130651fe265c3854, which resolves this thread.